ged2gwb/ged2gwb.camlp5

```diff
@@ -3671,10 +3671,12 @@ let set_undefined_death_interval s =
 (* Main *)
 
 let out_file = ref "a"
+let version = ref false
```

Comment:
I woul remove these `ref` and use `"-version", Arg.Unit Util.print_version_commit, "\n       Print version and commit numbers"`;